### PR TITLE
Component with svg icons

### PR DESCRIPTION
**Pull request by Horay**

### CODE REVIEW

File: src/theme/index.tsx [hunk 1]
```diff
@@ -45,6 +47,7 @@ export interface ThemeInterface {
   breakpoints: Breakpoints;
   zindex: ZIndex;
   [aliases: string]: any;
+  iconsize: IconSize;
```
Comment (janosio-nordcloud) [inline src/theme/index.tsx:50]: use `camelCase`

File: src/theme/index.tsx [hunk 2]
```diff
@@ -144,6 +147,14 @@ const theme: ThemeInterface = {
     xl: 1200,
   },
 
+  iconsize: {
```
Comment (janosio-nordcloud) [inline src/theme/index.tsx:150]: use camelCase

File: src/theme/index.tsx (hunk 1 above)
Comment (Horay) [reply src/theme/index.tsx]: fixed

File: src/theme/index.tsx (hunk 2 above)
Comment (Horay) [reply src/theme/index.tsx]: fixed

File: src/components/svgicon/SVGIcon.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import styled, {css} from "styled-components";
+import theme from "../../theme";
+import {getViewBox} from "../../utils/svgicons";
+import {getPath} from "../../utils/svgicons";
```
Comment (raczyk) [inline src/components/svgicon/SVGIcon.tsx:5]: Seems like this file has not been formatted with prettier.

File: src/components/svgicon/SVGIcon.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    `}
+    
+    ${({size}) =>
+    size && css`
+        width: ${theme.iconSize[size]};
+        height: ${theme.iconSize[size]};
+    `}
+`;
+
+export const SVGIcon: React.FC<SVGIconProps> = (props) => {
```
Comment (raczyk) [inline src/components/svgicon/SVGIcon.tsx:30]: You can use "destructuring" to get "name" from "props".

File: src/utils/svgicons.tsx [hunk 5]
```diff
@@ -0,0 +1,182 @@
+import React from "react";
+
```
Comment (raczyk) [inline src/utils/svgicons.tsx:2]: You can use prettier here as well.

File: src/stories/SWGIcon.stories.mdx [hunk 6]
```diff
@@ -0,0 +1,79 @@
+import { useState } from "react";
```
Comment (raczyk) [inline src/stories/SWGIcon.stories.mdx:1]: Name of the file is SWGIcon.stories.mdx should be "SVG...

File: src/utils/svgicons.tsx [hunk 7]
```diff
@@ -0,0 +1,182 @@
+import React from "react";
```
Comment (raczyk) [inline src/utils/svgicons.tsx:1]: You can use "camelCase" for naming the file.

File: src/components/svgicon/SVGIcon.tsx (hunk 4 above)
Comment (raczyk) [reply src/components/svgicon/SVGIcon.tsx:32]: `export const SVGIcon: React.FC<SVGIconProps> = ({ name, ...props }) => {`

and this to SVGIconProps 
`[x: string]: any;`

File: src/components/svgicon/SVGIcon.tsx (hunk 3 above)
Comment (Horay) [reply src/components/svgicon/SVGIcon.tsx]: done

File: src/components/svgicon/SVGIcon.tsx (hunk 4 above)
Comment (Horay) [reply src/components/svgicon/SVGIcon.tsx]: done

File: src/utils/svgicons.tsx (hunk 5 above)
Comment (Horay) [reply src/utils/svgicons.tsx:2]: done

File: src/stories/SWGIcon.stories.mdx (hunk 6 above)
Comment (Horay) [reply src/stories/SWGIcon.stories.mdx]: done

File: src/utils/svgicons.tsx (hunk 7 above)
Comment (Horay) [reply src/utils/svgicons.tsx:1]: done

**Pull request merged**